### PR TITLE
material ui example

### PR DESCRIPTION
**Pull request by krazyjakee**

### CODE REVIEW

File: examples/with-material-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Razzle x After.js
```
Comment (jaredpalmer) [inline examples/with-material-ui/README.md]: Can you provide a more descriptive README about the gist of the example?

File: examples/with-material-ui/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## How to use
+
+Download the example [or clone the whole project](https://github.com/jaredpalmer/razzle.git):
+
+```bash
+curl https://codeload.github.com/jaredpalmer/razzle/tar.gz/master | tar -xz --strip=2 razzle-master/examples/with-afterjs
```
Comment (jaredpalmer) [inline examples/with-material-ui/README.md]: Also fix the url to the code-load

**Comment by krazyjakee** [comment]
@jaredpalmer I updated the readme. Is it sufficient?

**Comment by krazyjakee** [comment]
Just a comment to say this is still waiting for feedback.

**Comment by krazyjakee** [comment]
Too much has changed. Also this PR had a major flaw with styling.

**Pull request closed without merging**